Jenkinsfile

```diff
@@ -106,15 +104,14 @@ try {
           }
         }
       }
-//    },
-//    'debian9': {
-//      node {
-//        sh 'setup_centreon_build.sh'
-//        sh "./centreon-build/jobs/web/${serie}/mon-web-unittest.sh debian9"
-//        junit 'ut.xml'
-//        if (currentBuild.result == 'UNSTABLE')
-//          currentBuild.result = 'FAILURE'
-//      }
+    },
```

Comment:
why paralellize centos8 jobs ? is it only a temporary job until centos8 integration is properly tested ?

Reply:
It is likely that CentOS 7 and CentOS 8 builds will run the same stages in parallel (same code, different environments).